code-challenges.test.js

```diff
@@ -52,5 +138,57 @@ const hand2 = [5, 5, 3, 3, 4]
 const hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
+describe('fullhouse', () => {
+    const hand1 = [5, 5, 5, 3, 3]
+    const hand2 = [5, 5, 3, 3, 4]
+    const hand3 = [5, 5, 5, 5, 4]
+    it('takes in an array of 5 numbers and determines whether or not the array is a “full house”.', () => {
+        expect(fullhouse(hand1)).toBeTruthy();
+        expect(fullhouse(hand2)).toBeFalsy();
+        expect(fullhouse(hand3)).toBeFalsy();
+    })
+})
 
 // b) Create the function that makes the test pass.
+
+// Pseudo:
+// create a function named "fullhouse" that takes in a parameter named "array"
+// create an if else statement to determine whether index "0" is equal to index "1" and "2" for the three of a kind
+// add an "&&" to determine if index "3" is equal to index "4" for the pair and return true if condition is met 
+// add an "or" to determine if index "0" is equal to index "2" and "3" for the three of a kind, add an "&&" to determine if index "1" is equal to index "4" for the pair and return true if condition is met 
+// add an "or" to determine if index "0" is equal to index "3" and "4" for the three of a kind, add an "&&" to determine if index "1" is equal to index "2" for the pair and return true if condition is met 
+// else return false
+
+const fullhouse = (array) => {
+
+    if (array[0] === array[1,2] && array[3] === array[4] || array[0] === array[2,3] && array[1] === array[4] || array[0] === array[3,4] && array[1] === array[2])
+        
+        return true
+
+    else{
+
+        return false
+
+    }
```

Comment:
My main feedback is whitespace issues. Code looks great!

Reply:
I'll keep that in mind. Thanks!